src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1579,12 +1596,18 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
 
         private async Task InvokeValidationEndpointAsync() {
             OpenIdConnectMessage request;
-
+            
+            // Note: The updated RFC standard says nothing about allowing GET requests beyond
```

Comment:
Okay, let's keep it for now then. I'll open a new ticket once this PR is merged to determine whether we should remove it or not.